composer.json

```diff
@@ -2,7 +2,8 @@
     "name": "andig/videodb",
     "type": "project",
     "require": {
-        "guzzlehttp/guzzle": "^6.3"
+        "guzzlehttp/guzzle": "^6.3",
+        "smarty/smarty": "v3.1.34"
```

Comment:
Do you need the specific version or rather ^3.1?

Reply:
have only tested with 3.1.34 as i believe that is the latest stable version

Reply:
don't need specific version ^3.1 should be ok.
my ide tool didnt allow for non specific entry.
apologies for confusion.

Reply:
Please update PR ;)

Reply:
composer.json updated and committed